x11-misc/j4-dmenu-desktop/metadata.xml

```diff
@@ -5,17 +5,22 @@
 		<email>sir.suriv@gmail.com</email>
 		<name>Diogo Pereira</name>
 	</maintainer>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
 	<maintainer type="project">
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
-	<longdescription lang="en">
-		j4-dmenu-desktop is a replacement for i3-dmenu-desktop. Its purpose is
-		to find .desktop files and offer you a menu to start an application
-		using dmenu. You can also execute shell commands using it.
+	<longdescription>
+		j4-dmenu-desktop is a replacement for i3-dmenu-desktop.
+		It's purpose is to find .desktop files and offer you a menu
```

Comment:
- Grammar: `Its` purpose vs. `It is` purpose
- ... unnecessary use of `you` ...

Reply:
Better now? :)

Reply:
`Its` was correct, but it is fine either way. ;)